tests/tensor/test_basic.py

```diff
@@ -1667,24 +1669,60 @@ def test_join_matrix_ints(self):
         assert (np.asarray(grad(s.sum(), b).eval()) == 0).all()
         assert (np.asarray(grad(s.sum(), a).eval()) == 0).all()
 
-    def test_join_matrix1_using_vertical_stack(self):
+    def test_join_matrix1_using_vstack(self):
         a = self.shared(np.array([[1, 2, 3], [4, 5, 6]], dtype=self.floatX))
         b = as_tensor_variable(np.array([[7, 8, 9]], dtype=self.floatX))
         c = as_tensor_variable(np.array([[9, 8, 7]], dtype=self.floatX))
-        s = vertical_stack(a, b, c)
+        s = vstack(a, b, c)
 
         want = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9], [9, 8, 7]])
         out = self.eval_outputs_and_check_join([s])
         assert (out == want).all()
 
-    def test_join_matrix1_using_horizontal_stack(self):
+    def test_join_matrix1_using_dstack(self):
```

Comment:
These tests are a bit too verbose IMO, they can be parameterized by shape (1d, 2d, 3d), function (hstack, vstack, dstack), and n_inputs (1,2,3) and just use `np.testing.assert_allclose` to make sure the pytensor version matches the numpy version